### PR TITLE
Updated dependencies

### PR DESCRIPTION
**Pull request by kukulich**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (Ocramius) [inline .github/dependabot.yml:1]: Thanks for adding dependabot! :+1:

File: .github/workflows/continuous-integration.yml [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+name: "Continuous Integration"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - "*.*.x"
+
+jobs:
```
Comment (Ocramius) [inline .github/workflows/continuous-integration.yml:11]: Not important but consider switching to something like https://github.com/laminas/laminas-ci-matrix-action/tree/6e706717c01e64ed1b1b1bbaa5041a0fbb1b3154#usage in future

It kinda explodes the matrix on its own, and with a lot of sensible stuff that I help maintaining :D

File: psalm.xml.dist [hunk 3]
```diff
@@ -3,8 +3,6 @@
     xmlns="https://getpsalm.org/schema/config"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"
-    totallyTyped="true"
```
Comment (Ocramius) [inline psalm.xml.dist:6]: `errorLevel="1"` should be used, I think

**Review by Ocramius — CHANGES_REQUESTED** [review]

File: psalm.xml.dist (hunk 3 above)
Comment (kukulich) [reply psalm.xml.dist:6]: Added and added `findUnusedPsalmSuppress="true"` as well.

Comment (Ocramius) [reply psalm.xml.dist:6]: Thanks!

**Review by Ocramius — APPROVED** [review]
:ship: thanks @kukulich!

**Pull request merged**